eslint.config.js

```diff
@@ -24,6 +24,9 @@ module.exports = {
     // This config updates the rule to require one or the other.
     'jsx-a11y/label-has-associated-control': [2, { assert: 'either' }],
     'react/destructuring-assignment': 'off',
+    // This updates the rule below to throw a warning rather than an error
+    // when using eslint-plugin-react 7.12.2 or above.
+    'react/jsx-wrap-multilines': 'warn',
```

Comment:
Is this for passivity? Will this rule be turned on in the future when the warnings are fixed?

Reply:
yes it should. it breaks current lint tests. We should get an issue logged to remove this rule